resources/options.js

```diff
@@ -160,4 +160,11 @@ document.addEventListener('DOMContentLoaded', restoreOptionsAvailability);
 $(document).ready(function() {
     addOptionClickEvents();
     changeListener();
+
+    //SideNav functionality
+    $('.button-collapse').sideNav({
+        edge: 'right',
+        closeOnClick: true
+    });
```

Comment:
This seems to be used in `options.js` as well as `about.js`. Can you move this to a separate file?